### ISSUE TITLE
sunshine stopped working with latest update

### ISSUE THREAD

**Issue by rcarmo** [ublue-os/bazzite#1260]
### Describe the bug

After the latest update, sunshine stopped working. Invoking it manually yields:

```
me@bazzite:~$ which sunshine
/usr/bin/sunshine
me@bazzite:~$ sunshine
sunshine: error while loading shared libraries: libboost_locale.so.1.81.0: cannot open shared object file: No such file or directory
```

### What did you expect to happen?

I expected sunshine to keep working

### Output of `rpm-ostree status`

```shell
rpm-ostree status
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-gnome:stable
                   Digest: sha256:02811976942b826cc69da8277dc6c381000cb115e12764fca2f186d416058519
                  Version: 40.20240621.0 (2024-06-21T05:11:51Z)
          LayeredPackages: fio python3.11 python3.11-devel sunshine
                Initramfs: '"-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf"' 

  ostree-unverified-registry:ghcr.io/ublue-os/bazzite-gnome:stable
                   Digest: sha256:02811976942b826cc69da8277dc6c381000cb115e12764fca2f186d416058519
                  Version: 40.20240621.0 (2024-06-21T05:11:51Z)
          LayeredPackages: fio python3.11 python3.11-devel sunshine
                Initramfs: '"-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf"' 
```

### Hardware

AMD Ryzen with iGPU

**Comment by KyleGospo** [comment]
@matte-schwartz could be a missing req in the spec?

**Comment by matte-schwartz** [comment]
Works fine here on F40, just installed and started on my OLED Deck without issue. 

I need to push a new version soon since they statically link boost now but there hasn't been an official release for it yet. Maybe try again after I push that update.

**Comment by rcarmo** [comment]
What is the image you are using?

**Comment by matte-schwartz** [comment]
I don't use bazzite, but it does not make sense to me why it would suddenly stop working for bazzite if workstation Fedora 40 still works.

**Comment by rcarmo** [comment]
Well, it doesn’t work on the Bazzite Gnome or KDE images (for deck or desktop). I’ve rebased to all four combinations and get the same error message. It working on Fedora 40 Workstation does not mean anything whatsoever…

**Comment by rcarmo** [comment]
OK, so I rebased to 20240531:

```
bazzite-rollback-helper rebase bazzite-deck-gnome:40-stable-20240531
```

...and it works, so this is definitely a release/image issue. Now to try to bisect this and see what revision broke it.

**Comment by rcarmo** [comment]
OK, so I've figured it out. It is both a release _and_ a `sunshine` COPR issue.

This release:

```
bazzite-rollback-helper rebase bazzite-deck-gnome:40-stable-20240616
```

...upgraded the `boost` libraries from 1.83.0-3.fc40 -> 1.83.0-5.fc40.

```
bazzite-rollback-helper rebase bazzite-deck-gnome:40-stable-20240614
```
...also changed something, but I can't pin it down.

Either of those breaks the COPR binary from @matte-schwartz. If that binary works on a fresh Fedora 40 install, that's likely because that install hasn't been updated yet.

```
bazzite-rollback-helper rebase bazzite-deck-gnome:40-stable-20240611
```

...works for me right now.

**Comment by HikariKnight** [comment]
> OK, so I've figured it out. It is both a release _and_ a `sunshine` COPR issue.
> 
> This release:
> 
> ```
> bazzite-rollback-helper rebase bazzite-deck-gnome:40-stable-20240616
> ```
> 
> ...upgraded the `boost` libraries from 1.83.0-3.fc40 -> 1.83.0-5.fc40.
> 
> ```
> bazzite-rollback-helper rebase bazzite-deck-gnome:40-stable-20240614
> ```
> 
> ...also changed something, but I can't pin it down.
> 
> Either of those breaks the COPR binary from @matte-schwartz. If that binary works on a fresh Fedora 40 install, that's likely because that install hasn't been updated yet.
> 
> ```
> bazzite-rollback-helper rebase bazzite-deck-gnome:40-stable-20240611
> ```
> 
> ...works for me right now.

I tested sunshine today on my Legion GO and it works fine.

**Comment by rcarmo** [comment]
Came back here to report that the sunshine service now core dumps upon startup on a fully updated system.

**Comment by HikariKnight** [comment]
uninstall sunshine and install it again using the ujust.
works fine for me

**Comment by rcarmo** [comment]
Nope. First thing I tried.

**Comment by rcarmo** [comment]
Yep. Still happening. Thanks for the summary, our future, hopefully benevolent, AI overlords.

**Comment by HikariKnight** [comment]
Still 0 people able to replicate, you are the only person with this issue.
wipe your sunshine config and start fresh

video of it working on a vm with bazzite-deck-nvidia as this is the only way i can test the nvidia images so this sees regular use
https://github.com/user-attachments/assets/d2748a32-e2ad-4dfd-b46a-9a270fbffc62

**Comment by noelmiller** [comment]
We are closing this issue since no one has been able to replicate it. Please do as @HikariKnight asks.
